int32 motorIndex
bool tripped